tests/Avalonia.Skia.UnitTests/TextLayoutTests.cs

```diff
@@ -506,6 +506,27 @@ public void Should_Include_Last_Line_When_Constraint_Is_Surpassed(string text, i
             }
         }
 
+        [InlineData("0123456789\r\n0123456789\r\n0123456789", 1, 1)]
```

Comment:
I think some test cases are redundant. You could change the test cases to cover this:

```
[InlineData("0123456789\r\n0123456789\r\n0123456789", 0, 3)]
[InlineData("0123456789\r\n0123456789\r\n0123456789", 4, 3)]
[InlineData("0123456789\r\n0123456789\r\n0123456789", 1, 1)]
```